packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md

```diff
@@ -6,6 +6,48 @@ title: Okta Identity Engine API Products release notes 2023
 
 > Help us improve our release notes by filling out this short [survey](https://surveys.okta.com/jfe/form/SV_4VEZcIGOX0TBgkC).
 
+## November
+
+### Monthly release 2023.11.0
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [SMS and voice support for OOB and MFA OOB grant flows is EA in Preview](#sms-and-voice-support-for-oob-and-mfa-oob-grant-flows-is-ea-in-preview) | November 8, 2023 |
+| [Org detail retrieval and user app listing for My Account API is GA in Preview](#org-detail-retrieval-and-user-app-listing-for-my-account-api-is-ga-in-preview) | November 8, 2023 |
+| [CORS restrictions removed for revoke endpoints](#cors-restrictions-removed-for-revoke-endpoints) | November 8, 2023 |
+| [Developer documentation update in 2023.011.0](#developer-documentation-update-in-2023.011.0) | November 8, 2023
+| [Bugs fixed in 2023.11.0](#bugs-fixed-in-2023-11-0) | November 8, 2023 |
+
+#### SMS and Voice support for OOB and MFA OOB grant flows is EA in Preview
+
+Direct authentication out-of-band (OOB) and multifactor out-of-band (MFA OOB) grant flows now support the phone authenticator with SMS and voice factors. See [Phone (MFA)](/docs/guides/configure-direct-auth-grants/fmfaoobsv/main/) and [Phone (primary factor)](/docs/guides/configure-direct-auth-grants/eoobsv/main/). <!-- OKTA-659932 -->
+
+#### Org detail retrieval and user app listing for My Account API is GA in Preview
+
+Two new endpoints for the My Account API (`/idp/myaccount/okta-applications` and `/idp/myaccount/organization`) allow customers to retrieve org details and get a full list of the current user's Okta apps for a particular end user. See [My Account API](/docs/api/openapi/okta-myaccount/guides/overview/). <!-- OKTA-651405, OKTA-663355 -->
```

Comment:
For links from the VuePress site to the redocly site we need to use the full URL.

I also switched around the order of the endpoints b/c it mirrors the structure of the rest of the sentence.

```suggestion
Two new endpoints for the My Account API (`/idp/myaccount/organization` and `/idp/myaccount/okta-applications`) allow customers to retrieve org details and get a full list of the current user's Okta apps for a particular end user. See [My Account API](https://developer.okta.com/docs/api/openapi/okta-myaccount/guides/overview/). <!-- OKTA-651405, OKTA-663355 -->
```